### ISSUE TITLE
Exif data is getting lost when opening an image from the photo library

### ISSUE THREAD

**Issue by HarelM** [apache/cordova-plugin-camera#452]
# Bug Report

## Problem
Exif data is getting lost when opening an image from the photo library on an android device.
This is in continue to #329.
Related to an issue reported by our users:
https://github.com/IsraelHikingMap/Site/issues/932

### What is expected to happen?
Exif data should be kept

### What does actually happen?
Exif data is getting lost

## Information
I've just tested the code using the instructions #415 and I still see an issue with exif data getting lost when getting images from the photo library on android device.
The code I'm using is the following:
navigator.camera.getPicture(() => ..., { destinationType: Camera.DestinationType.DATA_URL, sourceType: Camera.PictureSourceType.PHOTOLIBRARY, saveToPhotoAlbum: false}).
Code can be seen here:
https://github.com/IsraelHikingMap/Site/blob/d81cfdfd339060aab9b512352741b5af00c8a214/IsraelHiking.Web/sources/application/directives/image-capture.directive.ts#L41

### Environment, Platform, Device
Android, N2G47H 7.1.2, xiomi redmi 4

### Version information
Can be seen here:
https://github.com/IsraelHikingMap/Site/blob/master/IsraelHiking.Web/package.json
https://github.com/IsraelHikingMap/Site/blob/master/IsraelHiking.Web/config.xml

## Checklist

- [x] I searched for existing GitHub issues
- [ ] I updated all Cordova tooling to most recent version - I can't, latest cordova doesn't run on my windows 10 machine. the plugin is up-to-date though...
- [x] I included all the necessary information above

**Comment by HarelM** [comment]
Any updates on this? It's open for a while now...

**Comment by codeconsole** [comment]
I created a pull request with @ryaa 's working changes and missing commits from master:
https://github.com/apache/cordova-plugin-camera/pull/712

You can use the plugin in your code by running the following commands:
```
  cordova plugin remove cordova-plugin-camera
  cordova plugin add https://github.com/codeconsole/cordova-plugin-camera.git#ryaa
```


Can be verified working by running the following:
```
% cordova --version
10.0.0
```

```
  git clone https://github.com/codeconsole/exiftest
  cd exiftest
  cordova prepare
  cordova plugin remove cordova-plugin-camera
  cordova plugin add https://github.com/codeconsole/cordova-plugin-camera.git#ryaa
  cordova run ios
```

Choose a photo and notice all the exif information is there.

**Comment by HarelM** [comment]
Actually, I opened this bug for android, didn't know there was an issue with iOS...
I also think that largest version is working fine for android, but it's worth fixing this if iOS experience this issue...

**Comment by codeconsole** [comment]
yes, you are correct. Using my test app exiftest, I verified that the latest version 5.0.1 works on android.  It does not work on ios.
